### ISSUE TITLE
R4.0 installation

### ISSUE THREAD

**Issue by andrewjkwok** [JEFworks-Lab/veloviz#15]
Hi - A quick issue - I've been having problems installing veloviz with R4.0. However I'm getting messages saying that veloviz is not available for R4.0.0 regardless if I try installing via either bioconductor or `devtools::install_github`.

Is there any support for people not using R4.1?

**Comment by LylaAtta123** [comment]
Hi Andrew, 

The R4.1 requirement is for compatibility with bioconductor's latest release. That being said, neither veloviz nor any of its dependencies require R4.1 so you can download the package, change the R version requirement in the description file then build and install the package from the local copy (see below). 

Change DESCRIPTION file: 
`Depends:` 
`R (>= 4.0)`

Build package: (requires devtools and veloviz dependencies i.e. igraph, Rspectra.. see imports section of DESCRIPTION file):
`library(devtools)`
`devtools::build("veloviz-package", binary = FALSE, vignettes = FALSE)`

Install from source: 
`install.packages("veloviz_0.99.7.tar.gz",repos = NULL,type = "source")`

Let me know if you still have issues. 

Lyla

**Comment by andrewjkwok** [comment]
Hi Lyla - I'm afraid I'm not sure how exactly to execute this. I tried cloning the github repository into my R 4.0 path and changing the DESCRIPTION file, but how do I build or install the package after that (or is it not the correct thing to clone the github repo?)? Also, when I try `devtools::install.packages`, it gives me the error:

`Error: 'install.packages' is not an exported object from 'namespace:devtools'` 

I've never manually downloaded a package like this unfortunately so am a bit confused!

**Comment by LylaAtta123** [comment]
Hi Andrew, 

I downloaded the repo as a zipped folder and unzipped it in a local directory: 
<img width="385" alt="Screen Shot 2021-09-15 at 11 27 46 AM" src="https://user-images.githubusercontent.com/55152281/133463151-d705d409-0391-4b27-8cdb-d6ea9ded1d9b.png">

To build the package, I used `devtools::build` as follows: 
`library(devtools)`
`devtools::build([path to unzipped repo], binary = FALSE, vignettes = FALSE)`.
For me, because I'm working in the same directory that contains the repo folder this was just `devtools::build("veloviz-package", binary = FALSE, vignettes = FALSE)`

To install, I did the following (there was a mistake in my previous comment, this isn't a devtools function): 
`install.packages([path to package .tar.gz file],repos = NULL,type = "source")`. Again, if you're working in the same directory, it'll just be `install.packages("veloviz_0.99.7.tar.gz",repos = NULL,type = "source")`

Hope this works!
